Dockerfiles/Dockerfile.selenium-jenkins-python311

```diff
@@ -0,0 +1,18 @@
+FROM --platform=linux/amd64 python:3.11
```

Comment:
Not 100% certain about the 3.11-specific files. Do we need to do this? will it naturally work itself out over time?